pkg/expression/aggregation/avg.go

```diff
@@ -15,9 +15,9 @@
 package aggregation
 
 import (
+	"github.com/pingcap/tidb/pkg/expression"
```

Comment:
It's a little weird to depend on a package which is in the parent folder. But it seems that this package already unavoidably depends on `expression`, so I think it's fine.